_posts/dart-cuts.md

```diff
@@ -45,52 +44,52 @@ and finally
 
 > *"All 10 cities with highest percentage growth were outside transportation authority boundaries.  If we do not work together and turn this around, roadway funds will be lost to DART cities as well and as the RTC chases congestion, air quality and safety to the rural reaches of the region. This will reinforce even greater inefficient land use."*
 
-The cities reached out, NCTCOG started working on it, and in the meantime the cities do exactly the opposite of the current recommendations. Why ask for help in the first place?
+The cities had reached out for a study, NCTCOG started working on it, and in the meantime, the cities do exactly the opposite of the current recommendations. Why ask for help in the first place?
 
-In spite of this firm letter the cities continue on.
+In spite of this firm letter, the cities continue with the cuts.
```

Comment:
I think we need to say “support for cuts”, or “votes for cuts” or something similar since they can’t actually reduce the funding